_includes/start_in_docker/v2.0/mac-linux-steps.md

```diff
@@ -0,0 +1,167 @@
+## Before You Begin
+
+Make sure you have already [installed the official CockroachDB Docker image](install-cockroachdb.html).
+
+<!-- TODO: update the asciicast
+Also, feel free to watch this process in action before going through the steps yourself. Note that you can copy commands directly from the video, and you can use **<** and **>** to go back and forward.
+
+<asciinema-player class="asciinema-demo" src="asciicasts/start-a-local-cluster-docker.json" cols="107" speed="2" theme="monokai" poster="npt:0:43" title="Start a Local Cluster in Docker"></asciinema-player>
+-->
+
+## Step 1. Create a bridge network
+
+Since you'll be running multiple Docker containers on a single host, with one CockroachDB node per container, you need to create what Docker refers to as a [bridge network](https://docs.docker.com/engine/userguide/networking/#/a-bridge-network). The bridge network will enable the containers to communicate as a single cluster while keeping them isolated from external networks.
+
+~~~ shell
+$ docker network create -d bridge roachnet
+~~~
+
+We've used `roachnet` as the network name here and in subsequent steps, but feel free to give your network any name you like.
+
+## Step 2. Start the first node
+
+~~~ shell
+$ docker run -d \
+--name=roach1 \
+--hostname=roach1 \
+--net=roachnet \
+-p 26257:26257 -p 8080:8080  \
+-v "${PWD}/cockroach-data/roach1:/cockroach/cockroach-data"  \
+cockroachdb/cockroach-unstable start --insecure
```

Comment:
Is there a reason we aren't including the version tag here, a la `cockroachdb/cockroach-unstable:{{page.release_info.version}}`?

This comment applies to all of the examples.

Reply:
Done.